db/migrations/postgres/000004_create_txhistory_table.up.sql

```diff
@@ -0,0 +1,17 @@
+BEGIN;
+CREATE TABLE txhistory (
+  seq               SERIAL          PRIMARY KEY,
+  id                UUID            NOT NULL,
+  time              BIGINT          NOT NULL,
+  last_occurrence   BIGINT          NOT NULL,
+  tx_id             TEXT            NOT NULL,
+  status            TEXT            NOT NULL,
+  action            TEXT            NOT NULL,
+  count             INT             NOT NULL,
+  error             TEXT,
+  error_time        BIGINT,
+  info              TEXT
+);
+CREATE UNIQUE INDEX txhistory_id ON confirmations(id);
+CREATE INDEX txhistory_txid ON confirmations(tx_id);
```

Comment:
the unique indices here are using the wrong table

Reply:
This explains some poor performance I'd noticed in passing when compression was happening 👍

Reply:
solved now